app/commands/markdown/render_markdown.rb

```diff
@@ -0,0 +1,18 @@
+class Markdown::RenderMarkdown
```

Comment:
I wonder if it's worth merging this with `RenderDoc` and having the second argument passed as either `:doc` or `:text` instead? Unless you imagine them diverging later?